examples/tutorials/ordinary_model_example.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>You can run the notebook with with <code>warnings.filterwarnings("ignore")</code> (to suppress warnings and thus beautify the output), but remove it before saving (so a user can see warnings in his environment since some of them might be useful)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>